### PR TITLE
Fixed Setting up a cluster: oc tag doesn't work

### PR DESCRIPTION
**Pull request by mburke5678**

### CODE REVIEW

**Comment by mburke5678** [comment]
@miminar Can you take a look at my change based on https://github.com/openshift/origin/issues/8359 and https://github.com/openshift/openshift-docs/issues/2459 
Is this change still needed and if so, what OCP versions does this effect?

**Comment by miminar** [comment]
@mburke5678  thanks! This is right. Unless the pull spec is fully qualified (it contains a registry) or has at least 3 name components, it will be considered as an image stream. 

But I'd probably mention an alternative version:

```
oc tag docker.io/openshift/deployment-example:v2 deployment-example:latest
```

At least as a *note*.

**Comment by mburke5678** [comment]
@miminar PTAL. I added the alternative command as a note and moved the second note down paragraph and removed the NOTE to avoid having two notes so close together.

File: getting_started/administrators.adoc [hunk 1]
```diff
@@ -243,25 +243,34 @@ command created an image stream which tracks which images you wish to use.
...
 
 Your application's deployment config is watching `deployment-example:latest`
 and will trigger a new rolling deployment when the `latest` tag is updated
 to the value from `v2`.
 
+[NOTE]
+====
+You can also use an alternate version of the command:
+----
+oc tag --source=docker openshift/deployment-example:v2 deployment-example:latest
```
Comment (miminar) [inline getting_started/administrators.adoc]: `s!--source=docker openshift/deployment!docker.io/openshift/deployment!`

**Comment by mburke5678** [comment]
@miminar Made the change as suggested. PTAL

**Comment by mburke5678** [comment]
@jianlinliu Can you suggest someone in QE to review these changes? I apologize if this issue is not in your team's scope.

**Comment by jianlinliu** [comment]
@wanghaoran1988 this PR is relative to deployment, could you find someone to review these changes.

**Comment by wanghaoran1988** [comment]
LGFM

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

In 3.6, this file diverged from the 3.4 and 3.5 versions; I will make the same changes to that file in a separate PR.

File: getting_started/administrators.adoc [hunk 2]
```diff
@@ -243,25 +243,34 @@ command created an image stream which tracks which images you wish to use.
...
 
 Your application's deployment config is watching `deployment-example:latest`
 and will trigger a new rolling deployment when the `latest` tag is updated
 to the value from `v2`.
 
+[NOTE]
+====
+You can also use an alternate version of the command:
+----
+oc tag docker.io/openshift/deployment-example:v2 deployment-example:latest
```
Comment (ahardin-rh) [inline getting_started/administrators.adoc]: $ before the command

File: getting_started/administrators.adoc [hunk 3]
```diff
@@ -243,25 +243,34 @@ command created an image stream which tracks which images you wish to use.
...
-NOTE: For this next step we'll need to ensure that Docker is able to pull images
-from the host system. Ensure you have completed the instructions about setting the
-`--insecure-registry` flag from xref:../install_config/install/host_preparation.adoc#install-config-install-host-preparation[Host Preparation].
-
 As a developer, building new container images is as important as deploying them.
 {product-title} provides tools for running builds as well as building
 source code from within predefined builder images via the Source-to-Image
 toolchain.
 
+For this procedure, make sure that Docker is able to pull images
```
Comment (ahardin-rh) [inline getting_started/administrators.adoc]: s/make sure/ensure (just a nit)

**Comment by ahardin-rh** [comment]
@mburke5678 Just 2 minor comments from me. Otherwise, looks good!

**Comment by vikram-redhat** [comment]
@mburke5678  - the commits aren't squashed. Please make sure to squash commits.

**Comment by vikram-redhat** [comment]
@mburke5678 - make sure to squash commits.

**Comment by mburke5678** [comment]
[rev_history]
|xref: getting_started/administrators.adoc#try-it-out[Advanced Installation]
|Changed the `tag` command to mark a new image as being desired for deployment.
%

**Comment by ahardin-rh** [comment]
The revision history here needs edited. There's an extra space, it's missing some formatting, and the path is not correct. It should be:
xref:../getting_started/administrators.adoc#getting-started-administrators[Administrators: Setting Up a Cluster]

**Comment by vikram-redhat** [comment]
@mburke5678 - this also has multiple commits. Please remember to squash.

**Pull request merged**